crates/walrus-service/src/node/event_blob_writer.rs

```diff
@@ -1332,14 +1348,15 @@ impl EventBlobWriter {
             .latest_attested_checkpoint_sequence_number
             .set(checkpoint_sequence_number.try_into()?);
 
+        let node_capability_object_id = self.node.node_capability();
         match self
             .node
             .contract_service
             .certify_event_blob(
                 metadata.try_into()?,
                 checkpoint_sequence_number,
                 self.current_epoch,
-                self.node.node_capability(),
+                node_capability_object_id,
```

Comment:
Remark: This is an unrelated change, correct?

Reply:
yes, its minor but i can put it back.